apicast-gateway/apicast.yml

```diff
@@ -105,7 +114,7 @@ objects:
 parameters:
 - name: AMP_RELEASE
   description: "AMP release tag."
-  value: 2.1.0-CR2-redhat-1
+  value: "2.0.0.GA"
```

Comment:
Shouldn't this be 2.0.1.GA ?

Reply:
done in dedce9bb0089fb28c3993c590908fa07e885c47a